### ISSUE TITLE
Can We Edit Multimodal Large Language Models?

### ISSUE THREAD

**Issue by JH-Yin** [zjunlp/EasyEdit#115]
Hello, I am very interested in your multimodal editing work(Can We Edit Multimodal Large Language Models?) published in EMNLP2023, but I have some questions while reading the code. When evaluating the editing effect, you use the method of combining prompt and target, and feeding them into the forward function to obtain logits, in order to obtain accuracy. Why choose this way to evaluate? Why not just use prompt and input it into the model.generate() function to obtain prediction results for evaluation.

**Comment by cheng-siyuan** [comment]
> Hello, I am very interested in your multimodal editing work(Can We Edit Multimodal Large Language Models?) published in EMNLP2023, but I have some questions while reading the code. When evaluating the editing effect, you use the method of combining prompt and target, and feeding them into the forward function to obtain logits, in order to obtain accuracy. Why choose this way to evaluate? Why not just use prompt and input it into the model.generate() function to obtain prediction results for evaluation.

Hello, thank you for your interest in our work. Using the method of combining prompt and target can be more convenient for us to evaluate the metrics. On the other hand, we have noticed that the native generate() function of a model like OPT does not support the input of multimodal image vector features, so for convenience we are using the method of combining prompt and target.

**Comment by JH-Yin** [comment]
Thank you very much for your answer.

I have another question. In this work, you proposed five indicators, namely Reliability, T-Locality, M-Locality, T-Generality, and M-Generality. However, in the main results of the paper, you only reported four indicators (Reliability, T-Locality, M-Locality, T-Generality). Although M-Generality is shown in Figure 5, numerical values cannot be directly obtained, and the results shown in the figure are also incomplete. May I ask if you will disclose the complete experimental results for future research and comparison?

**Comment by cheng-siyuan** [comment]
> Thank you very much for your answer.
> 
> I have another question. In this work, you proposed five indicators, namely Reliability, T-Locality, M-Locality, T-Generality, and M-Generality. However, in the main results of the paper, you only reported four indicators (Reliability, T-Locality, M-Locality, T-Generality). Although M-Generality is shown in Figure 5, numerical values cannot be directly obtained, and the results shown in the figure are also incomplete. May I ask if you will disclose the complete experimental results for future research and comparison?

We did not show M-Generality in the main table, primarily for cross-comparison of Locality and Generality. We are also aware of this problem and we will give specific values in the Readme as soon as possible.

**Comment by zxlzr** [comment]
Hi, do you any further questions?

**Comment by JH-Yin** [comment]
Thank you very much for your answer and I look forward to seeing the complete experimental results.

I have another question about SERAC. During testing, the input_text which is consisting of prompt and label will be input to the model. SERAC will first classify the input_text. Will this not lead to label leakage?

**Comment by tbozhong** [comment]
The classifier is used solely for determining the direction of the input flow, and there is no concept of labels involved. Therefore, there is no possibility of label leakage.

**Comment by JH-Yin** [comment]
Thank you very much for your patient answer.

Maybe my previous statement was unclear. What I want to ask is when using SERAC for multi-modal editing, what text is used for classification? Is it input_text (prompt+target)?

**Comment by tbozhong** [comment]
Yes, your understanding is correct.

**Comment by JH-Yin** [comment]
So input_text (prompt+target) will also be used for classification during the testing phase? Is this approach reasonable? Will there be a gap with real-world applications, because in reality there may only be prompts but no targets?

**Comment by tbozhong** [comment]
Currently, all model editing methods require `targets` to explicitly determine the direction of the edits. Without using `targets`, it becomes challenging for the editing methods to ascertain the correct editing direction.

**Comment by zxlzr** [comment]
Hi, do you have further questions?

**Comment by JH-Yin** [comment]
Thank you very much for your patient answer. I have no more questions.
